internal/config/attestation.go

```diff
@@ -11,11 +11,74 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
+	"sort"
 
 	"github.com/edgelesssys/constellation/v2/internal/attestation/measurements"
+	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 	"github.com/edgelesssys/constellation/v2/internal/variant"
 )
 
+const (
+	AttestationTypeAWSNitroTPM        AttestationType = "aws-nitro-tpm"        // AttestationTypeAWSNitroTPM
+	AttestationTypeAzureSEVSNP        AttestationType = "azure-sev-snp"        // AttestationTypeAzureSEVSNP
+	AttestationTypeAzureTrustedLaunch AttestationType = "azure-trusted-launch" // AttestationTypeAzureTrustedLaunch
+	AttestationTypeGCPSEVES           AttestationType = "gcp-sev-es"           // AttestationTypeGCPSEVES
+	AttestationTypeQEMUVTPM           AttestationType = "qemu-vtpm"            // AttestationTypeQEMUVTPM
+	AttestationTypeDefault            AttestationType = "default"              // AttestationTypeDefault is the default attestation type for the cloud provider
+)
```

Comment:
We already have the [`variant`](https://github.com/edgelesssys/constellation/blob/main/internal/variant/variant.go) package, which basically does the same things `AttestationType` does.
imo, we should remove `AttestationType` in favor of using the types from `variant` to avoid duplication and having to change code in multiple places if we want to change a single thing

Reply:
I agree it's better and made the change. But should we rename the package to `attestation_variant`? Variant is indeed unclear naming as @katexochen mentioned